com.ibm.streamsx.topology/com.ibm.streamsx.topology.python/namespace-info.spl

```diff
@@ -238,18 +238,95 @@ The functions must be builtins or in a separate module to the module building
 the topology. The functions must not be dynamically created. The modules
 containing the function are copied into the Streams Application Bundle (sab file).
 
+# Newly supported items
```

Comment:
I might remove this line, the documentation is 'new' until when? Maybe something like
```
# Stream methods
```
instead.